packages/components/psammead-timestamp/README.md

```diff
@@ -22,12 +22,20 @@ When a date or time is to be displayed inline inside a paragraph.
 | Argument   | Type   | Required | Default | Example                  |
 |------------|--------|----------|---------|--------------------------|
 | `datetime` | string ([supported formats](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time#Valid_datetime_Values)) | Yes | N/A | `2006-09-24T05:00-07:00` |
+| `typographyStyle` | string | Yes | N/A | `font-size: 1rem` |
 
 ## Usage
 
-```js
+```jsx
+import { latin } from '@bbc/gel-foundations/scripts';
+import { getPica } from '@bbc/gel-foundations/typography';
+
+const typographyStyle = getPica(latin);
```

Comment:
Im curious of others opinions on this,

I imagined passing in the function `get*` (e.g. getGreatPrimer) to be used in psammead-timestamp itself. Means we can also default to `getPica` internally for usability

thoughts @pjlee11 @sareh @DenisBBC?

Reply:
Are you saying something like:
```
  <Timestamp datetime="2019-03-01T14:00+00:00" typographyFunc={getPica} script={latin}>
    Friday 1 March 2019
  </Timestamp>
```

Reply:
Yeah, with an internal proptypes default of `getPica`

Reply:
That would keep inline with the other components which currently internally have the `script` mocked in their tests/stories. Feels like a more consistent approach